core/src/main/kotlin/com/imashnake/animite/core/ui/Stats.kt

```diff
@@ -0,0 +1,30 @@
+package com.imashnake.animite.core.ui
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+
+@Composable
+fun <T> StatsRow(
+    stats: List<T>,
+    modifier: Modifier = Modifier,
+    content: @Composable (T) -> Unit
+) {
+    Row(
```

Comment:
How many "stats" are we expecting, and how does this scale to different screen sizes?

Reply:
Three for the media page and profile screen, tested it extensively for the media page so it should be fine for now 🫡